### ISSUE TITLE
[EDITOR] North Qingyun Peak Teleporter Fix

### ISSUE THREAD

**Issue by EliteMasterEric** [EliteMasterEric/Teyvat.moe#218]
The following data was generated by this user using the Editor mode. Any required images should be uploaded to this issue by dragging and dropping.
```
{
  "name": {
    "en": "North Qingyun Peak Teleporter Fix"
  },
  "cluster": false,
  "icons": {
    "filter": "none",
    "base": {
      "marker": true
    },
    "done": {
      "marker": true
    }
  },
  "data": [
    {
      "geometry": {
        "type": "Point",
        "coordinates": [
          -41.12580528903429,
          23.868932285286984
        ]
      },
      "type": "Feature",
      "properties": {
        "popupTitle": {
          "en": "Teleporter Update"
        },
        "popupContent": {
          "en": "North Qingyun Peak"
        },
        "popupMedia": ""
      },
      "id": 1
    }
  ]
}
```

**Comment by EliteMasterEric** [comment]
This should help with #191.

**Comment by EliteMasterEric** [comment]
Fixed in v0.9.5.
